plugin.json

```diff
@@ -113,6 +113,14 @@
                 "help_text": "Display subscription name in post when a subscription posts to a channel",
                 "placeholder": "",
                 "default": false
+            },
+            {
+                "key": "AdminAPIToken",
+                "display_name": "Admin API Token",
+                "type": "text",
+                "help_text": "This token is used to fetch some extra details for a Jira issue for comment related and 'issue created' subscription events.",
```

Comment:
Also, this isn't matching with the description of `what to test?` Shouldn't we update it to something like `Set this token to view notifications when the user is not connected?

We should add the note here as mentioned in the description: `API token is preferred to be created using an admin Jira account. Otherwise, notification will not be delivered for the project the API token user does not have access to.`

Also, can we add this somewhere in readme or link here itself, on how to generate this token?

Reply:
@ayusht2810 Updated the help test. We will add documentation for this PR once the PR is approved